cluster/saltbase/salt/monit/kube-proxy

```diff
@@ -3,7 +3,9 @@ group kube-proxy
 start program = "/etc/init.d/kube-proxy start"
 stop program = "/etc/init.d/kube-proxy stop"
 if does not exist then restart
-if failed port 10249
-  protocol HTTP request "/healthz"
-  with timeout 10 seconds
-  then restart
+if failed
```

Comment:
It's a perfectly reasonable question, and I would have been concerned if no one had asked :)